### ISSUE TITLE
No timeout on requests

### ISSUE THREAD

**Issue by Tatermen** [brontide/unifiapi#2]
After a couple of days, I found that my Python script had hung while connecting to the Unifi controller. I believe this is because there is no timeout in the requests() call, per the [Requests documentation](http://docs.python-requests.org/en/master/user/quickstart/#timeouts):

> You can tell Requests to stop waiting for a response after a given number of seconds with the timeout parameter. Nearly all production code should use this parameter in nearly all requests. Failure to do so can cause your program to hang indefinitely

**Comment by brontide** [comment]
Good call, should be easy to add.  The main calls are just wrappers around calls to requests and extending them to add a timeout should be easy enough.
